openedx/core/djangoapps/ccxcon/tests/test_api.py

```diff
@@ -0,0 +1,188 @@
+"""
+Unit tests for the API module
+"""
+
+import datetime
+import mock
+import pytz
+import urlparse
+from nose.plugins.attrib import attr
+
+from opaque_keys.edx.keys import CourseKey
+from student.tests.factories import AdminFactory
+from xmodule.modulestore.django import modulestore
+from xmodule.modulestore.tests.django_utils import (
+    ModuleStoreTestCase,
+    SharedModuleStoreTestCase,
+    TEST_DATA_SPLIT_MODULESTORE)
+from xmodule.modulestore.tests.factories import (
+    CourseFactory,
+    ItemFactory,
+)
+
+from openedx.core.djangoapps.ccxcon import api as ccxconapi
+from .factories import CcxConFactory
+
+
+def flatten(seq):
+    """
+    For [[1, 2], [3, 4]] returns [1, 2, 3, 4].  Does not recurse.
+    """
+    return [x for sub in seq for x in sub]
```

Comment:
actually I do not think list comprehension allows any parenthesis in this specific case